apps/webservice/src/app/[workspaceSlug]/_components/CreateSystem.tsx

```diff
@@ -79,23 +89,23 @@ export const CreateSystemDialog: React.FC<{
             <DialogHeader>
               <DialogTitle>New System</DialogTitle>
               <DialogDescription>
-                Systems are a group of processes, releases, and runbooks for an
+                Systems are a group of processes, releases, and runbooks for
                 applications or services.
               </DialogDescription>
             </DialogHeader>
-
             <FormField
               control={form.control}
               name="name"
               render={({ field }) => (
                 <FormItem>
-                  <FormLabel>Name</FormLabel>
+                  <FormLabel>System Name</FormLabel>
```

Comment:
Its styled create system? Why do we need to set name here, I won't include it adds more clutter  to the ui